### PR TITLE
Makefile: add explicit rule for commands/mancontent_gen.go

### PR DESCRIPTION
**Pull request by jj1bdx**
* The newly built mancontent_gen.go must be under the ${GOPATH} source tree
* This solves the issues in https://github.com/git-lfs/git-lfs/issues/3159

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -140,6 +140,12 @@ BUILD_TARGETS = bin/git-lfs-darwin-amd64 bin/git-lfs-darwin-386 \
 	bin/git-lfs-freebsd-amd64 bin/git-lfs-freebsd-386 \
 	bin/git-lfs-windows-amd64.exe bin/git-lfs-windows-386.exe
 
+.PHONY : mangen
+mangen : $(GOPATH)/src/github.com/git-lfs/git-lfs/commands/mancontent_gen.go
+
+$(GOPATH)/src/github.com/git-lfs/git-lfs/commands/mancontent_gen.go :
```
Comment (ttaylorr) [inline Makefile]: This rule look great! I had two quick notes.

Firstly, could you change this line from the fully-qualified GOPATH-style to just `commands/mancontent_gen.go`? Make will interpret this correctly, and it matches the style of other recipes in this file.

Secondly, could you please add documentation about what this recipe does? You can look to other recipes in this file for inspiration about what is standard practice here. Thanks.

**Comment by ttaylorr** [comment]
Thanks again! I hope you don't mind, I pushed a few commits of my own to your branch and merged it in.

**Comment by jj1bdx** [comment]
@ttaylorr Sorry for getting late. Your fixes looked better than mine. Appreciated. :+1:

**Comment by ttaylorr** [comment]
> @ttaylorr Sorry for getting late.

Not to worry at all; I forgot how quick it had been since I commented, so I think it was I who should have waited longer. Thank you again for your wonderful contribution! 😄

**Pull request merged**